data/scrape/tsv_summary.tsv

```diff
@@ -11,10 +11,10 @@ ale_latn_broad.tsv	ale	Aleut	Aleut	Latin		False	Broad	True	104
 grc_grek_broad.tsv	grc	Ancient Greek (to 1453)	Ancient Greek	Greek		False	Broad	True	90222
 ara_arab_broad.tsv	ara	Arabic	Arabic	Arabic		False	Broad	False	7279
 arc_hebr_broad.tsv	arc	Imperial Aramaic (700-300 BCE); Official Aramaic (700-300 BCE)	Aramaic	Hebrew		False	Broad	False	1156
+arm_armn_e_narrow.tsv	arm	Armenian	Armenian	Armenian	Eastern Armenian, standard	False	Narrow	True	14182
 arm_armn_w_narrow.tsv	arm	Armenian	Armenian	Armenian	Western Armenian, standard	False	Narrow	True	14065
 arm_armn_w_narrow_filtered.tsv	arm	Armenian	Armenian	Armenian	Western Armenian, standard	True	Narrow	True	14040
 arm_armn_e_narrow_filtered.tsv	arm	Armenian	Armenian	Armenian	Eastern Armenian, standard	True	Narrow	True	14177
```

Comment:
Very minor thing, but maybe we just do finer sorting than just Wiktionary name and transcription level, since it gives  us a random sorting of the four Armenian files (all of which have the same Wiktionary name and transcription level) and of other similar cases.

Reply:
+1

Reply:
I changed it to sort on wiki name, then whether or not they're filtered (just so that they are grouped, not really necessary but maybe looks nice), and on the path -- which has to be unique and contains transcription level anyway.